package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "postcss-calc",
-  "version": "5.3.1",
+  "version": "6.0.0",
```

Comment:
Can you remove the version bump from here please? https://github.com/sindresorhus/np takes care of this. 😄